### ISSUE TITLE
Updates to the Accessibility Dimension

### ISSUE THREAD

**Issue by mvmaltitz** [worldbank/GEEST#83]
### Accessibility Dimension
The Accessibility Dimension (AD) evaluates women’s daily mobility by examining their access to essential services. Levels of enablement for work access in this dimension are determined by service areas, which represent the geographic zones that facilities like childcare, supermarkets, universities, banks, and clinics can serve based on proximity. The nearer these facilities are to where women live, the more supportive and enabling the environment becomes for their participation in the workforce.
The WEE Accessibility score is computed as the average of the factors presented in the table below. Each factor is assessed through service areas as derived from network analyses around key facilities - this defines varying levels of access. After each subfactor analysis, the territory is divided into a grid, and the factor score is calculated as the average score within each raster cell.

<img width="633" height="775" alt="Image" src="https://github.com/user-attachments/assets/8235ef56-f5b2-4fde-ae2e-d351f05226ab" />

<img width="639" height="347" alt="Image" src="https://github.com/user-attachments/assets/f0dc08f5-0eaf-4c64-b69c-6f0adb1ff964" />

### Tasks:

- [ ] Update the national level thresholds
- [ ] Implement the local thresholds
- [ ] WEE score should be 45% (Recommended)
- [ ] Put some links into the user interface of where to fetch data (OSM or Humdata)
- [ ] Include OSM down-loader

### Meeting Comments:

1. User can choose to download OSM data or upload their own datasets
2. The user does not need to automatically download the Humdata 

**End Date:** 2026-02-25
